CGATPipelines/PipelineGeneset.py

```diff
@@ -523,14 +523,16 @@ def buildCDSFasta(infiles, outfile):
 
     tmpfile = P.getTempFile(".")
 
+    transcript_table = P.toTable(infile_transcripts)
+
     dbhandle = sqlite3.connect(PARAMS["database_name"])
     cc = dbhandle.cursor()
+    statement = '''SELECT DISTINCT protein_id, transcript_id
+                   FROM %(transcript_table)s''' % locals()
+
     tmpfile.write("protein_id\ttranscript_id\n")
     tmpfile.write("\n".join(
-        ["%s\t%s" % x for x in
-         cc.execute(
-             "SELECT DISTINCT protein_id, transcript_id "
-             "FROM transcript_info")]))
```

Comment:
Many thanks Steve!

Although it looks good to me, I would appreciate a second opinion from @CGATOxford/contributors 

e.g. regarding this line, changing the select statement with a parametrised table is the way forward but please comment that somewhere so everyone is aware about the introduced change that might cause backwards compatibility problems with legacy code.